html/renderer.go

```diff
@@ -3,6 +3,7 @@ package html
 import (
 	"bytes"
 	"fmt"
+	"github.com/gomarkdown/markdown/internal/uri"
```

Comment:
this needs a gofmt prolly

Reply:
`gofmt -s -w .` fixes formatting in files outside of this PR. Feel free to suggest different options.

Reply:
ah, let me send a PR for the whole project then